tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/AddressBuilder.cs

```diff
@@ -25,20 +25,20 @@ private AddressBuilder()
             _country = "Country";
         }
 
-        internal AddressBuilder WithLine1(string line1)
+        internal static AddressBuilder Create()
```

Comment:
I really don't see the benefit of this in these classes

Reply:
This is to limit the surface area of the `AddressBuilder` class, creating one focused method to create a new instance. Also it keeps the rest of the code consistent with the fluent syntax.